package.json

```diff
@@ -2,14 +2,11 @@
 	"private": true,
 	"scripts": {
 		"build": "tsc -b",
-		"watch": "tsc -b -w",
-		"prerelease": "npm run build",
-		"release": "lerna publish --exact --force-publish --yes --sync-workspace-lock",
-		"release:next": "npm run release -- --dist-tag next"
+		"watch": "tsc -b -w"
 	},
 	"devDependencies": {
-		"@lerna-lite/cli": "latest",
-		"@lerna-lite/publish": "latest",
+		"@changesets/changelog-github": "^0.4.8",
```

Comment:
Can we install changesets packages globally while running CI? Since no need to use it locally, removing it from package.json can shrink node_modules a bit.

Reply:
Changesets provides a CLI for creating changeset entries. This is run locally. Maybe I should provide a small contributing document where this is explained.

Reply:
Got it!